### PR TITLE
Fixes for proper handling test in CVP pipeline.

### PR DESCRIPTION
**Pull request by phracek**
Fixes for proper handling test in CVP pipeline.

* correct detection of docker address
* print 'oc status --suggest' for getting logs
* delete proper objects in CVP pipeline
* Tag image in CVP pipeline to proper version
* fix calling path in test-openshift.yml

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

File: test-lib-openshift.sh [hunk 1]
```diff
@@ -112,8 +112,14 @@ function ct_os_run_in_pod() {
 # Arguments: service_name - name of the service
 function ct_os_get_service_ip() {
   local service_name="${1}" ; shift
+  ocp_docker_address='172\.30\.[0-9\.]*'
```
Comment (zmiklank) [inline test-lib-openshift.sh:115]: I would prefer to set `ocp_docker_address` as a local variable.

File: test-lib-openshift.sh [hunk 2]
```diff
@@ -112,8 +112,14 @@ function ct_os_run_in_pod() {
...
+  if [ "${CVP:-0}" -eq "1" ]; then
```
Comment (zmiklank) [inline test-lib-openshift.sh:116]: AFAIK global variables should be mentioned in the function header as an explanatory comment. But I can see, that this is not a rule.

File: test-lib-openshift.sh [hunk 3]
```diff
@@ -392,7 +399,13 @@ function ct_os_delete_project() {
 # Deletes all objects within the project.
 # Handy when we have one project and want to run more tests.
 function ct_delete_all_objects() {
-  for x in bc builds dc is isimage istag po pv pvc rc routes secrets svc ; do
+  objects="bc builds dc is isimage istag po pv pvc rc routes secrets svc"
```
Comment (zmiklank) [inline test-lib-openshift.sh:402]: I would prefer to set `objects` as a local variable.

File: test-openshift.yaml [hunk 4]
```diff
@@ -51,7 +51,12 @@
           shell: oc project {{ openshift_project_name }}
 
         - name: Import the image into OpenShift
-          shell: oc import-image {{ image_name }} --from={{ image_full_name }} --insecure=true --confirm
+          shell: oc import-image {{ image_name }}:{{ environment[0]['VERSION'] }} --from={{ image_full_name }} --insecure=true --confirm
+          retries: 3
+          delay: 10
+
+        - name: Tag image into OpenShift
+          shell: oc tag {{ image_name }}:{{ environment[0]['VERSION'] }} nodejs:{{ environment[0]['VERSION'] }}
```
Comment (zmiklank) [inline test-openshift.yaml:59]: Do we want to tag every container image as `nodejs`?

File: test-openshift.yaml [hunk 5]
```diff
@@ -68,4 +73,4 @@
 
         # Run tests on OpenShift 4
         - name: Run a sclorg test suite in OpenShift 4
-          shell: VERSION={{ environment[0]['VERSION'] }} IMAGE_NAME={{ image_name }} OS={{ environment[0]['OS'] }} CVP=1 bash test/run-openshift-remote-cluster  | tee {{ cvp_artifacts_dir }}/{{ image_name }}.log
+          shell: VERSION={{ environment[0]['VERSION'] }} IMAGE_NAME={{ image_name }} OS={{ environment[0]['OS'] }} CVP=1 bash ./run-openshift-remote-cluster  | tee {{ cvp_artifacts_dir }}/{{ image_name }}.log
```
Comment (zmiklank) [inline test-openshift.yaml:76]: The `run-openshift-remote-cluster` is in `test` directory, why this change?

**Review by zmiklank — CHANGES_REQUESTED** [review]
Thanks for the PR, please see my comments.

Would you please consider adding the PR description into the commit-msg?
Moreover, would it be possible for me to see the CVP tests logs, to double-check the changes?
Without it, I can not really set LGTM.

File: test-lib-openshift.sh (hunk 1 above)
Comment (phracek) [reply test-lib-openshift.sh:115]: Make sense

File: test-lib-openshift.sh (hunk 2 above)
Comment (phracek) [reply test-lib-openshift.sh:116]: Good point.

File: test-openshift.yaml (hunk 5 above)
Comment (phracek) [reply test-openshift.yaml:76]: I will clarify it, what is working directly

File: test-openshift.yaml (hunk 4 above)
Comment (phracek) [reply test-openshift.yaml:59]: This is a great catch. Definitely not.

File: test-openshift.yaml (hunk 5 above)
Comment (phracek) [reply test-openshift.yaml]: Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83

File: test-openshift.yaml (hunk 4 above)
Comment (phracek) [reply test-openshift.yaml]: Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83

File: test-lib-openshift.sh (hunk 3 above)
Comment (phracek) [reply test-lib-openshift.sh]: ﻿﻿﻿Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83

File: test-lib-openshift.sh (hunk 2 above)
Comment (phracek) [reply test-lib-openshift.sh:116]: Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83

File: test-lib-openshift.sh (hunk 1 above)
Comment (phracek) [reply test-lib-openshift.sh]: Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83

**Comment by phracek** [comment]
[test-all]

**Review by zmiklank — APPROVED** [review]
